packages/frontend/src/utils/getWalletURL.js

```diff
@@ -11,3 +12,16 @@ export default (https = true) => {
 
     return `${https ? 'https://' : ''}wallet.${networkName}near.org`;
 };
+
+const getMyNearWalletUrl = (https = true) => {
+    const prefix = {
+        'testnet': 'testnet.',
```

Comment:
Can we use the actual values from `Environments` as keys here please? That way we're future proofed against typos.

Reply:
Great catch 👍